packages/libpostal/.local.vimrc

```diff
@@ -0,0 +1,6 @@
+" https://github.com/thinca/vim-localrc/blob/master/doc/localrc.txt
```

Comment:
Maybe this file should be better placed into root directory of the repositorty (there already is a `.vscode` directory).

Reply:
Ooops. There is already one.
It seems that this file, in the libpostal package is no more useful...

Reply:
thank you, I just deleted it in the master branch